app/views/ems_container/ad_hoc/_chart_view_form.html.haml

```diff
@@ -26,5 +26,17 @@
                                          "options"            => "dash.dateOptions",
                                          "date"               => "dash.timeFilter.date"}
 
+    %dropdown.dropdown-kebab-pf
```

Comment:
As per https://github.com/angular-ui/bootstrap/wiki/Migration-guide-for-prefixes, please use `uib-dropdown`, if possible.